website/src/components/discourse/index.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+import feedStyles from './styles.module.css';
+
+export default function DiscourseFeed({
+  order = 'latest_topic',
+  status = undefined,
+  after = undefined,
+  before = undefined,
+  inString = undefined, 
+  min_posts = undefined,
+  max_posts = undefined,
+  min_views = undefined,
+  max_views = undefined,
+  tags = undefined, 
+  term = undefined,
+  category = undefined,
+  title = undefined,
+  link_text = "See latest topics",
+  link_href = "https://discourse.getdbt.com/",
```

Comment:
As the default, it would be good for the CTA to be `Ask the Community` which goes to `https://discourse.getdbt.com/new-topic?category=XXXX&tags=comma,separated`

Note that in search you can either use `,` in tags for any of and `+` for all of, so you'd need to replace +s with ,s here too

https://meta.discourse.org/t/how-to-create-a-new-topic-link-with-pre-filled-information/28074

Then we could override it on the forum homepage to be the Read More ones you currently have. (Side note: are we allowed to set them to null/undefined to cause no button to be shown?)